crates/astria-sequencer-types/src/sequencer_block_data.rs

```diff
@@ -82,10 +98,12 @@ impl SequencerBlockData {
         let Some(data_hash) = header.data_hash else {
             bail!(Error::MissingDataHash);
         };
-        action_tree_root_inclusion_proof
-            .verify(action_tree_root.as_bytes(), data_hash)
-            .wrap_err("failed to verify action tree root inclusion proof")?;
-        // TODO(https://github.com/astriaorg/astria/issues/270): also verify last_commit_hash
+        if !cfg!(feature = "relayer-test") {
```

Comment:
pls remove, we should not modify the code for tests 

you can see the tests in this file for how to construct this correctly

Reply:
sure, added this like a means to keep up with the rapidly changing main branch. needed some control parameter to stay constant so I could keep chaining the functionality adding code in this branch. will fix this as last thing in this pr.